src/anyvar/restapi/schema.py

```diff
@@ -26,38 +26,31 @@ class DependencyInfo(BaseModel):
 
 
 class InfoResponse(BaseModel):
-    """Describe response for the /info endpoint"""
+    """Describe response for the GET /info endpoint"""
+
+    model_config = ConfigDict(
+        json_schema_extra={
+            "examples": [
+                {
+                    "anyvar": {"version": "0.1.2.dev58+g81eb592.d20230316"},
+                    "ga4gh_vrs": {"version": "0.7.6"},
```

Comment:
This is unrelated to this PR, but I'm kind of wondering if we should change this to `ga4gh_vrs_python` so it's clear that this is not the VRS version?